### PR TITLE
Slider: Update active state color and updating semantic slots

### PR DESCRIPTION
**Pull request by marygans**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Update active pressed state color for slider background-color and thumb to match design toolkit spec. Updated palette references to semantic colors

**Before:**
slider-before

**After:**
slider-after

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9792)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts [hunk 1]
```diff
@@ -65,6 +74,15 @@ export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
     }
   };
 
+  const slideBoxHoverThumbStyles = !disabled && {
+    border: `2px solid ${sliderHoverSectionColor}`,
+    selectors: {
+      [HighContrastSelector]: {
```
Comment (Vitalius1) [inline packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts]: nit: I see these HighContrast selector repeated several times. Maybe worth extracting into a const and reuse?

File: packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts [hunk 2]
```diff
@@ -22,21 +22,22 @@ const GlobalClassNames = {
...
 
   // Tokens
-  const sliderInteractedActiveSectionColor = palette.themePrimary;
+  const sliderInteractedActiveSectionColor = semanticColors.inputBackgroundCheckedHovered;
+  const sliderHoverSectionColor = semanticColors.inputBackgroundChecked;
   const sliderInteractedInactiveSectionColor = palette.themeLight;
-  const sliderRestActiveSectionColor = palette.neutralSecondary;
-  const sliderRestInactiveSectionColor = palette.neutralTertiaryAlt;
+  const sliderRestActiveSectionColor = semanticColors.smallInputBorder;
+  const sliderRestInactiveSectionColor = semanticColors.buttonBackgroundChecked;
```
Comment (Vitalius1) [inline packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts]: This const says it’s in rest state, but the semantic slot says it’s checked?

File: packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts [hunk 3]
```diff
@@ -22,21 +22,22 @@ const GlobalClassNames = {
...
 export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
   const { className, titleLabelClassName, theme, vertical, disabled, showTransitions, showValue } = props;
-  const { palette } = theme;
+  const { semanticColors, palette } = theme;
   const classNames = getGlobalClassNames(GlobalClassNames, theme);
 
   // Tokens
-  const sliderInteractedActiveSectionColor = palette.themePrimary;
+  const sliderInteractedActiveSectionColor = semanticColors.inputBackgroundCheckedHovered;
+  const sliderHoverSectionColor = semanticColors.inputBackgroundChecked;
```
Comment (Vitalius1) [inline packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts]: hover const vs checked slot… is there anything else that relates to hover that you can use from semantic slots?

**Comment by dzearing** [comment]
@marygans friendly ping here; any changes pending?

**Pull request closed without merging**